### ISSUE TITLE
Wrong install paths for CMake files

### ISSUE THREAD

**Issue by GoogleCodeExporter** [OGRECave/ogre-procedural#63]
```
The current install target will, among other things which are correct, install 
these files to these locations:

-- Installing: /tmp/ogre-procedural/pkg/PROCEDURAL/cmake/FindOIS.cmake
-- Installing: /tmp/ogre-procedural/pkg/PROCEDURAL/cmake/FindOGRE.cmake
-- Installing: /tmp/ogre-procedural/pkg/PROCEDURAL/cmake/FindPROCEDURAL.cmake
-- Installing: /tmp/ogre-procedural/pkg/PROCEDURAL/cmake/FindPkgMacros.cmake
-- Installing: /tmp/ogre-procedural/pkg/PROCEDURAL/cmake/PreprocessorUtils.cmake
-- Installing: /tmp/ogre-procedural/pkg/PROCEDURAL/cmake/ProceduralMacros.cmake

This has multiple problems. First of all, you should really only install 
FindPROCEDURAL.cmake because installing any of the others would lead to 
duplicates as they are already installed by ogre.

Next, your install path is all wrong. CMake already has a convenient constant 
for you to always get its correct and respective install path (CMAKE_ROOT). Why 
not use it?

In conclusion, CMake/CMakeLists.txt should now only contain (and I mean *only*, 
as in nothing else):

install(FILES Packages/FindPROCEDURAL.cmake
    DESTINATION ${CMAKE_ROOT}/Modules)

and nothing else. If you really want me to I can make a patch but this should 
be enough to make all changes.
```

Original issue reported on code.google.com by `svenst...@gmail.com` on 24 Jul 2011 at 8:11
